Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -112,9 +122,10 @@ bool PlatformerObjectBehavior::UpdateProperty(
     behaviorContent.SetAttribute("ignoreDefaultControls", (value == "0"));
   else if (name == _("Can grab platform ledges"))
     behaviorContent.SetAttribute("canGrabPlatforms", (value == "1"));
-  else if (name == _("Automatically grab platform ledges without having to move horizontally"))
+  else if (name == _("Automatically grab platform ledges without having to "
+                     "move horizontally"))
     behaviorContent.SetAttribute("canGrabWithoutMoving", (value == "1"));
-  else if (name == _("Use frame per second dependent trajectories (deprecated)"))
+  else if (name == "useLegacyTrajectory")
     behaviorContent.SetAttribute("useLegacyTrajectory", (value == "1"));
   else if (name == _("Can go down from jumpthru platforms"))
```

Comment:
Why this change?

Reply:
Good practice to start using short identifier for property names, and labels for what to show to the user. Apart from clarity, this also ensure the label is only at one place in the code (I forgot to update the other and so the property update was broken without any warning).